projects/canopy/src/styles/variables/components/_hero.scss

```diff
@@ -2,9 +2,9 @@
   --hero-padding: var(--space-sm);
   --hero-color: var(--color-white);
   --hero-bg-color: var(--color-super-blue-dark);
-  --hero-overlay-bg-color: var(--color-black);
   --hero-rule-color: rgba(255, 255, 255, 0.2);
   --hero-bg-alpha: 0.1;
+  --hero-overlay-bg-color: rgba(0, 0, 0, var(--hero-bg-alpha));
```

Comment:
I don't see any overlay now so maybe we can rename this to ```---hero-card-notification-bg-color``` ?

Reply:
Good point @HarshaDewani

Reply:
Yea, I thought about that but it's still like an overlay bg for those components. In hero header it was also used as an overlay over the component, that is why I left it as overlay :)

Reply:
I am happy to keep it as it is given the fact changing css vars name can cause breaking changes.